### ISSUE TITLE
[MAUI][.NET8] recipeSearch project builds fail to compile when use 8.0.0-nightly.9076+sha.f0db96bd41-azdo.8316708.

### ISSUE THREAD

**Issue by Zhanglirong-Winnie** [dotnet/maui#17055]
### Description

When update  Microsoft.Maui.Controls Version to 8.0.0-nightly.9076+sha.f0db96bd41-azdo.8316708. RecipeSearch project builds fail and have several errors.

### Steps to Reproduce

1. Add this feed to your local NuGet.config:
![dailymauifeed2023-08-29](https://github.com/dotnet/maui/assets/77434865/5078e1b9-0c95-4c94-a87b-cd1b278c7c89)
2. Download recipeSearch from https://github.com/rachelkang/recipeSearch.
3. Update Microsoft.Maui.Controls Version to 8.0.0-nightly.9076+sha.f0db96bd41-azdo.8316708.
4. Then right click solution and select 'Restore NuGet Packages'.
5. Click Build Recipes button by right click project.
**Mac platform:**
![dailymauiissue2023-08-29](https://github.com/dotnet/maui/assets/77434865/7991233d-5eb5-47b5-b457-5b83a8510b0f)
**Windows platform:**
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0103	The name 'InitializeComponent' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\App.xaml.cs	10	Active
Error	CS0103	The name 'InitializeComponent' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\AppShell.xaml.cs	10	Active
Error	CS0103	The name 'InitializeComponent' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\EditRecipePage.xaml.cs	9	Active
Error	CS0103	The name 'ratingLabel' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\EditRecipePage.xaml.cs	23	Active
Error	CS0103	The name 'InitializeComponent' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\MyRecipesPage.xaml.cs	14	Active
Error	CS0103	The name 'vMyRecipesListView' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\MyRecipesPage.xaml.cs	40	Active
Error	CS0103	The name 'vMyRecipesListView' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\MyRecipesPage.xaml.cs	41	Active
Error	CS0103	The name 'vMyRecipesListView' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\MyRecipesPage.xaml.cs	45	Active
Error	CS0103	The name 'vMyRecipesListView' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\MyRecipesPage.xaml.cs	46	Active
Error	CS0103	The name 'InitializeComponent' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\NewRecipePage.xaml.cs	12	Active
Error	CS0103	The name 'ratingLabel' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\NewRecipePage.xaml.cs	26	Active
Error	CS0103	The name 'InitializeComponent' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\RecipeDetailPage.xaml.cs	11	Active
Error	CS0103	The name 'InitializeComponent' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\SearchResultDetailPage.xaml.cs	14	Active
Error	CS0103	The name 'InitializeComponent' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\SearchResultsPage.xaml.cs	11	Active
Error	CS0103	The name 'searchBar' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\SearchResultsPage.xaml.cs	13	Active
Error	CS0103	The name 'InitializeComponent' does not exist in the current context	Recipes (net8.0-android)	C:\Users\v-paulwang\Downloads\recipeSearch-main (4)\recipeSearch-main\WhatToEat\Views\StartingPage.xaml.cs	11	Active
Error		Improper project configuration: no AppxManifest is specified, but WindowsPackageType is not set to MSIX.	Recipes	C:\Users\v-paulwang\.nuget\packages\microsoft.windowsappsdk\1.3.230602002\buildTransitive\Microsoft.Build.Msix.Packaging.targets	148	
Error		One or more duplicate file names were detected.  All image output filenames must be unique: search (Resources\Images\search.png), cookbook (Resources\Images\cookbook.png)	Recipes	C:\Users\v-paulwang\.nuget\packages\microsoft.maui.resizetizer\8.0.0-nightly.9076\buildTransitive\Microsoft.Maui.Resizetizer.After.targets	595	

**Log:**
Mac:
[daily maui.zip](https://github.com/dotnet/maui/files/12462620/daily.maui.zip)
Windows:
[windowsdailymaui.zip](https://github.com/dotnet/maui/files/12462628/windowsdailymaui.zip)

### Version with bug
Maui: 8.0.0-preview.7.8842
SDK: 8.0.100-preview.7.23376.3
Microsoft.Maui.Controls: 8.0.0-nightly.9076+sha.f0db96bd41-azdo.8316708

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

8.0.0-preview.7.8842

### Affected platforms

iOS, Android, Windows, macOS

### Affected platform versions
Windows: Windows 11 Enterprise N 22H2
Mac: Ventura 13.5

**Comment by rachelkang** [comment]
Hi, @Zhanglirong-Winnie - I'm seeing you mentioned both Maui: 8.0.100-preview.7.23376.3 and Microsoft.Maui.Controls: 8.0.0-nightly.9076+sha.f0db96bd41-azdo.8316708 in the issue. You should have the nightly build installed for Maui as well. Could you please clarify where each of these versions come into play so we can help you get unblocked?

**Comment by samhouts** [comment]
Related? https://github.com/dotnet/maui/issues/17040

**Comment by Zhanglirong-Winnie** [comment]
> Hi, @Zhanglirong-Winnie - I'm seeing you mentioned both Maui: 8.0.100-preview.7.23376.3 and Microsoft.Maui.Controls: 8.0.0-nightly.9076+sha.f0db96bd41-azdo.8316708 in the issue. You should have the nightly build installed for Maui as well. Could you please clarify where each of these versions come into play so we can help you get unblocked?

I'm sorry for the confusion caused by my error message. I have updated the version information.
Maui: 8.0.0-preview.7.8842
SDK: 8.0.100-preview.7.23376.3
Microsoft.Maui.Controls: 8.0.0-nightly.9076+sha.f0db96bd41-azdo.8316708

**Comment by CathyZhu0110** [comment]
> Hi, @Zhanglirong-Winnie - I'm seeing you mentioned both Maui: 8.0.100-preview.7.23376.3 and Microsoft.Maui.Controls: 8.0.0-nightly.9076+sha.f0db96bd41-azdo.8316708 in the issue. You should have the nightly build installed for Maui as well. Could you please clarify where each of these versions come into play so we can help you get unblocked?

Hi @rachelkang , we just followed the process https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/35041/Daily-MAUI-Testing, and installed the latest 17.8 P2 main build, and updated the latest Microsoft.Maui.Controls to the latest nightly build. So the .NET SDK and MAUI version is 8.0 P7, and Microsoft.Maui.Controls is 8.0.0-nightly.9076+sha.f0db96bd41-azdo.8316708.

**Comment by mattleibow** [comment]
Probably a duplicate of https://github.com/dotnet/maui/issues/17040

But, the duplicate images are also interesting... Can you attach a binlog? https://github.com/xamarin/xamarin-macios/wiki/Diagnosis

Or, try a newer nightly as soon as it comes out. I tested with the build that should become the nightly in a few hours (8.0.0-ci.net8.9114) and it seems to work.

**Comment by Zhanglirong-Winnie** [comment]
> Probably a duplicate of #17040
> 
> But, the duplicate images are also interesting... Can you attach a binlog? https://github.com/xamarin/xamarin-macios/wiki/Diagnosis
> 
> Or, try a newer nightly as soon as it comes out. I tested with the build that should become the nightly in a few hours (8.0.0-ci.net8.9114) and it seems to work.
[binlogs.zip](https://github.com/dotnet/maui/files/12483314/binlogs.zip)
For the duplicate images. I attached binlog.
